src/OfficeToolPlus/Dictionaries/Languages/tr-tr.xaml

```diff
@@ -669,11 +669,11 @@
     <sys:String x:Key="Settings_Download">İndir</sys:String>
     <sys:String x:Key="Settings_Download_Ask_Where_To_Save">Dosyaların nereye kaydedileceğini sor</sys:String>
     <sys:String x:Key="Settings_Download_Ask_Where_To_Save.Description">Office'i indirmeden önce Office yükleme dosyalarının nereye kaydedeceğileceğini seçin.</sys:String>
-    <sys:String x:Key="Settings_Download_MsFileList_API">Get file list using Microsoft's API</sys:String>
-    <sys:String x:Key="Settings_Download_MsFileList_API.Description">If enabled, the file list will be obtained from Microsoft's API, otherwise the file list will be generated using built-in rules.</sys:String>
+    <sys:String x:Key="Settings_Download_MsFileList_API">Microsoft'un API'sini kullanarak dosya listesini alın</sys:String>
+    <sys:String x:Key="Settings_Download_MsFileList_API.Description">Etkinleştirilirse dosya listesi Microsoft'un API'sinden alınacaktır, aksi takdirde dosya listesi yerleşik kurallar kullanılarak oluşturulacaktır.</sys:String>
     <sys:String x:Key="Settings_Download_Engine">İndirme Aracı</sys:String>
     <sys:String x:Key="Settings_Download_Engine.Description">Office'i indirirken hangi indirme aracının kullanılacağını seçin.</sys:String>
-    <sys:String x:Key="Settings_Download_Engine.Thunder">Thunder</sys:String>
+    <sys:String x:Key="Settings_Download_Engine.Thunder">İndirme Aracı</sys:String>
```

Comment:
You are translating `Thunder` to `İndirme Aracı` and `İndirme Aracı` is same as the **Settings_Download_Engine** content. I don't suggested you to do that. Please check and edit it if possible.